apps/dashboard/app/views/files/edit.html.erb

```diff
@@ -1,5 +1,239 @@
+<div class="d-flex flex-row flex-grow-1">
+  <div class="flex-sm-column card mx-2">
```

Comment:
Dont' know how big your screen is - but this renders very large for me (half the page) - I'm guessing because the filename is so long.


![image](https://github.com/OSC/ondemand/assets/4874123/14188543-2e1a-4e61-b432-c010ce2a0daf)